tests/e2e/specs/out-of-game/snackbar.spec.js

```diff
@@ -0,0 +1,47 @@
+import { myUser } from '../../fixtures/userFixtures';
+import en from '../../../../src/translations/en.json';
+
+describe('Lobby - Page Content of ranked game', () => {
+  beforeEach(() => {
+    setup();
+  });
+
+  it('Changes games type. snackbar is visible', () => {
+    //switch to ranked
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').should('exist');
+    
+  });
+
+  it('Changes games type. check appropriate text', () => {
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} ${en.global.ranked}`);
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} ${en.global.casual}`);

```

Comment:
Unless we're specifically testing translations I think it's best to use the literal expected text in english since it doesn't have potential gotchas around missed keys in the json
```suggestion
    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} Ranked`);
    clickGameModeSwitch();
    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} Casual`);

```